Makefile

```diff
@@ -418,24 +418,17 @@ build-volumes:
 	$(SILENT)docker volume inspect $(GOPATH_VOLUME_NAME) >/dev/null 2>&1 || docker volume create $(GOPATH_VOLUME_NAME)
 	$(SILENT)docker volume inspect $(GOCACHE_VOLUME_NAME) >/dev/null 2>&1 || docker volume create $(GOCACHE_VOLUME_NAME)
 
-.PHONY: main-builder-image
-main-builder-image: build-volumes
-	@echo "+ $@"
-	$(SILENT)# Ensure that the go version in the image matches the expected version
-	# If the next line fails, you need to update the go version in rox-ci-image/images/stackrox-build.Dockerfile
-	grep -q "$(shell head -n 1 EXPECTED_GO_VERSION)" <(docker run --rm "$(BUILD_IMAGE)" go version)
```

Comment:
Besides checking `EXPECTED_GO_VERSION`, this target `main-builder-image` called `build-volumes` through a declared dependency. It's fine if you remove `main-builder-image` but in places that depended on it you should put `build-volumes` instead.